com.vrcfury.vrcfury/Editor/VF/Service/HapticAnimContactsService.cs

```diff
@@ -111,8 +112,8 @@ string name
                 if (cache.TryGetValue(allowSelf, out var cached)) return cached;
 
                 var prefix = $"{name}/Anim{(allowSelf ? "" : "Others")}";
-                var maxDist = Math.Max(0, actions.Max(a => Math.Max(a.startDistance, a.endDistance)));
-                var minDist = Math.Min(0, actions.Min(a => Math.Min(a.startDistance, a.endDistance)));
+                var maxDist = Math.Max(0, actions.Max(a => Math.Max(a.startDistance, a.endDistance))) * (worldScale ? 1f : socketOwner.transform.lossyScale.z);
```

Comment:
probably better to use animRoot rather than socketOwner, since animRoot is what actually holds the contacts. Although in practice they're probably the same.